content/fr/containerization.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contenérisation
+status: Completed
+category: Technology
+tags: ["application", "", ""]
+---
+
+## Ce que c'est
+
+La contenérisation est l'action de regrouper une application et ses dépendances dans une [image de conteneur](/container-image/).
+Le processus de construction du conteneur nécessaire de respecter le standard de l'[Open Container Initiative](https://opencontainers.org) (OCI).
+Du moment que le résultat est une image qui respecte ce standard, l'outil de contenérisation qui sera utilisé n'a pas d'importance.
+
+## Problème auquel il répond
+
+Avant que les conteneurs ne se généralisent, les organisations orchestraient de nombreuses applications au sein d'une unique [machine physique](/bare-metal-machine/).
```

Comment:
La traduction me parait pas bonne : 

"Before containers became prevalent, organizations relied on virtual machines (VMs) to orchestrate multiple applications on a single [bare-metal machine](https://glossary.cncf.io/bare-metal-machine/)."

J'aurais plutôt traduit ça comme ça : 

Avant que les conteneurs prévalent, les organisations s'appuyaient sur des machines virtuelles pour orchestrer différentes applications sur une machine physique.